content/en/blog/2024/ambient-reaches-beta/index.md

```diff
@@ -0,0 +1,116 @@
+with appropriate cautions.  This is a huge milestone for Istio, bringing both Layer 4 and Layer 7 mesh features to production
+readiness without sidecars.
+
+## Why ambient mode?
+
+We listened to the feedback from Istio users and observed a growing demand for mesh capabilities for their applications but who found the resource overhead and operational complexity of sidecars hard to overcome.
+functions for their applications and found the complexity and overhead of sidecars from service meshes hard to overcome.
+Istio’s ambient mode has been designed for simplified operations without requiring changes or restarts to your application
+alleviating the previous barriers of complexity faced by users looking to implement service mesh.
+
+In ambient mode it is trivial to add or remove applications from the mesh. You can now simply label your namespace with `istio.io/dataplane-mode=ambient and all applications in the namespace are added to the mesh. This immediately secures all traffic with mTLS, all without sidecars or the need to restart applications.
+your namespace with `istio.io/dataplane-mode=ambient`, and your application pods in the namespace are added to the mesh and the
+communications among your application pods in ambient are secured with mTLS, without restartarting or the need for sidecars!
+
+Some challenges that Istio sidecar users have shared with us include: how Istio can break applications after sidecars are
+added, the large consumption of resources by sidecars, and the inconvenience of the requirement to restart application pods
+with every new proxy release.
+
+As a community, we listened and designed ambient mode to tackle these problems. This new feature from Istio was named ambient
+mode as it was designed to be transparent to your application, ensuring no additional configuration was required to adopt it
+and required no restarting of applications by users. You can refer to the [introducing ambient mesh blog](/blog/2022/introducing-ambient-mesh/)
+for more information on why we started ambient mode in Istio.
+
+## How does ambient mode make adoption easier?
+
+Istio’s ambient mode introduces lightweight, shared node proxies and optional Layer 7 (L7) proxies and removes the need for
+traditional sidecar proxies from the data plane. The core innovation behind ambient mode is that it slices the L4 and L7
+processing into two distinct layers. This layered approach allows you to adopt Istio incrementally, enabling a smooth
+transition from no mesh, to a secure overlay (L4), to optional full L7 processing — on a per-namespace basis, as needed across
+your fleet.
+
+Ambient mode works without any modification required to your existing Kubernetes deployments. Users can label a namespace to
+add all of its workloads to Istio’s ambient mode, or opt-out of certain deployments as needed. By utilizing ambient mode, users
```

Comment:
opt-out certain deployments as needed.   (remove the 'of')

Reply:
updated